tests/components/knx/test_config_flow.py

```diff
@@ -167,14 +167,16 @@ async def test_tunneling_setup_for_multiple_found_gateways(hass: HomeAssistant)
         "homeassistant.components.knx.async_setup_entry",
         return_value=True,
     ) as mock_setup_entry:
-        manuel_tunnel_flow = await hass.config_entries.flow.async_configure(
+        manual_tunnel_flow = await hass.config_entries.flow.async_configure(
             manual_tunnel["flow_id"],
             {
                 CONF_HOST: "192.168.0.1",
                 CONF_PORT: 3675,
             },
         )
-        assert manuel_tunnel_flow["data"] == {
+        await hass.async_block_till_done()
```

Comment:
There are three more places where we create an entry but don't block till done that I see.

Reply:
Sorry, fixed